addons/mrp/models/mrp_production.py

```diff
@@ -1877,7 +1877,7 @@ def _default_amounts(production):
                 ml_vals = initial_move._prepare_move_line_vals()
                 backorder_move_to_ignore = backorder_moves[-1] if has_backorder_to_ignore[initial_move.raw_material_production_id] else self.env['stock.move']
                 for move in (initial_move + backorder_moves - backorder_move_to_ignore):
-                    if not initial_move.move_line_ids:
+                    if not move.move_line_ids and (initial_move.product_uom_qty >= initial_move.quantity):
```

Comment:
A test was failing without this condition because the consumed qty was fully set and marked as picked on the original MO rather than on the back orders if I remember correctly. I am updating the diff and pushing again to remember which test but the use case seemde legit to me.